### DISCUSSION TITLE
Nominate @joshuatcasey as a contributor

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#135] [PHP Team]
I'd like to nominate @joshuatcasey as a PHP contributor. He has been contributing to the PHP language family rewrite, specifically in the `composer` side of things:

https://github.com/paketo-buildpacks/composer/pull/1
https://github.com/paketo-buildpacks/rfcs/pull/170
https://github.com/paketo-buildpacks/composer-install/pull/2

He has implemented features and contributed to a number of thoughtful discussions on the subject of composer. He would make a fantastic contributor with solid context on an important piece of the language family.

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/php-maintainers

**Comment by sophiewigmore** [comment]
bump @thitch97

**Comment by joshuatcasey** [comment]
Looks like we have 2/3 of the @paketo-buildpacks/php-maintainers in agreement, can we push this forward?

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/steering-committee we have a super majority, can we make Josh an official contributor?

**Comment by ryanmoran** [comment]
Congrats @joshuatcasey! You are now a PHP Contributor.
